scripts/west_commands/runners/__init__.py

```diff
@@ -2,34 +2,54 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+import importlib
+import logging
+
 from runners.core import ZephyrBinaryRunner, MissingProgram
 
+_logger = logging.getLogger('runners')
+
+def _import_runner_module(runner_name):
+    try:
+        importlib.import_module(f'runners.{runner_name}')
+    except ImportError:
+        # Runners are supposed to gracefully handle failures when they
+        # import anything outside of stdlib, but they sometimes do
+        # not. Catch ImportError to handle this.
+        _logger.warning(f'The module for runner "{runner_name}" '
+                        'could not be imported. This most likely means '
+                        'it is not handling its dependencies properly. '
+                        'Please report this to the zephyr developers.')
```

Comment:
This shows the name of the (broken) runner but does not name the broken import. Fixed in follow-up PR #33448